src/components/Dialog/Dialog.module.scss

```diff
@@ -2,19 +2,11 @@
 
 $arrow-size: 12px;
 
-.dialog-node {
-  position: absolute;
-  top: 0;
-  height: 0;
-  z-index: 100000;
-}
-
-.monday-style-tooltip-arrow {
+:global(.monday-style-tooltip-arrow) {
```

Comment:
why here it's global?

Reply:
I've put it by mistake - thought it's overriding smth. Fixed